generators/client/templates/angular/package.json.ejs

```diff
@@ -27,24 +27,25 @@
     "node_modules"
   ],
   "dependencies": {
-    "@angular/common": "7.2.6",
-    "@angular/compiler": "7.2.6",
-    "@angular/core": "7.2.6",
-    "@angular/forms": "7.2.6",
-    "@angular/platform-browser": "7.2.6",
-    "@angular/platform-browser-dynamic": "7.2.6",
-    "@angular/router": "7.2.6",
+    "@angular/common": "7.2.7",
+    "@angular/compiler": "7.2.7",
+    "@angular/core": "7.2.7",
+    "@angular/forms": "7.2.7",
+    "@angular/platform-browser": "7.2.7",
+    "@angular/platform-browser-dynamic": "7.2.7",
+    "@angular/router": "7.2.7",
     "@fortawesome/angular-fontawesome": "0.3.0",
     "@fortawesome/fontawesome-svg-core": "1.2.15",
     "@fortawesome/free-solid-svg-icons": "5.7.2",
     "@ng-bootstrap/ng-bootstrap": "4.0.4",
+    "@nguniversal/express-engine": "7.1.0",
```

Comment:
Why is this added?

Reply:
Oh sorry, forgot to comment about this one. It is a peer dependency of `ngx-cookie` and needs to be added manually. See also here https://github.com/salemdar/ngx-cookie/issues/78. This has changed in this commit: https://github.com/salemdar/ngx-cookie/commit/1956eff8940859147268f5b2b6cc3e7619a89247. Alternatively we can revert back to the previous version

Reply:
I don't understand why we should add this... The main purpose of this dependency is to do server-side rendering

Reply:
You are right. It should not be needed. Let me check again

Reply:
So you were right, it was not needed at all. I removed it. Lets wait for the CI again

For some reason it was failing locally and the error message was the same as the one described here https://github.com/salemdar/ngx-cookie/issues/78 and hastily I applied one of the workarounds without checking what the dep is actually for. Good thing we have review :)

Reply:
@wmarques So it appears that the problem is that webpack fails to build the bundle as can be seen at the CI logs after commit cf2bfad.Tried to silence webpack by using the [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/) in commit 8115e8c but this is no good at runtime :(. So we either wait for a fix for ngx-cookie or revert back to 4.0.2, except if you have some better idea/solution to this problem. What do you think?

Reply:
Let's revert to 4.0.2 and wait for a fix. If it's not fixed in few weeks we can think about dropping this, I don't link being dependent on a non-maintained lib...

Reply:
:+1: